packages/xrpl/package.json

```diff
@@ -46,13 +55,13 @@
     "build:browserTests": "webpack --config ./test/webpack.config.js",
     "analyze": "run-s build:web --analyze",
     "watch": "run-s build:lib --watch",
-    "clean": "rm -rf dist",
+    "clean": "rm -rf dist build coverage",
     "docgen": "typedoc && echo js.xrpl.org >> ../../docs/CNAME",
     "prepublish": "run-s clean build",
-    "test": "nyc mocha --config=test/.mocharc.json --exit",
-    "test:integration": "TS_NODE_PROJECT=tsconfig.build.json nyc mocha ./test/integration/**/*.test.ts ./test/integration/*.test.ts",
-    "test:browser": "npm run build:browserTests && TS_NODE_PROJECT=tsconfig.build.json nyc mocha ./test/browser/*.ts",
-    "test:watch": "TS_NODE_PROJECT=src/tsconfig.json mocha --config=test/.mocharc.json --watch --reporter dot",
+    "test": "jest --verbose false --silent=false --runInBand ./test/**/*.test.ts --testPathIgnorePatterns=./test/integration --testPathIgnorePatterns=./test/fixtures",
```

Comment:
I'm curious whether the unit tests can be run in parallel

Reply:
Fixed, yes they can